jina/drivers/rank/aggregate/__init__.py

```diff
@@ -68,6 +70,62 @@ def _insert_query_matches(
                     m.chunks.append(chunk_matches_by_id[match_chunk_id])
             query.matches.append(m)
 
+    @staticmethod
+    def _group_by(match_idx, col_name):
+        # sort by ``col
+        _sorted_m = np.sort(match_idx, order=col_name)
+        _, _doc_counts = np.unique(_sorted_m[col_name], return_counts=True)
+        # group by ``col``
+        return np.split(_sorted_m, np.cumsum(_doc_counts))[:-1]
+
+    @staticmethod
+    def _sort_doc_by_score(r):
+        """
+        Sort a list of (``doc_id``, ``score``) tuples by the ``score``.
```

Comment:
can you add an empty line between method description and it's parameter section in docstring? It's not reported an error because it's an internal method.

Reply:
can u suggrst the change inline?